src/components/layouts/LayoutContainer.tsx

```diff
@@ -25,6 +25,8 @@ export const LayoutContainer = ({
     const { pageConfig } = usePageConfig();
     const { descriptor, path, type, config = {} } = layoutProps;
 
+    console.log(`Desc=${descriptor} Type=${type}`);
+
     const bem = BEM(type);
```

Comment:
skal Bem fortsatt brukes i denne fila?

Reply:
Ja, det må den for å håndtere ulike layouts som kommer som props til komponenten